### ISSUE TITLE
Can't move SimpleBar in IE11

### ISSUE THREAD

**Issue by dolin5** [Grsmto/simplebar#118]
I'm attempting to add SimpleBar to an [ESRI Layer List widget](https://github.com/Esri/arcgis-dijit-layer-list). Everything is working great in Chrome and Firefox, but in IE11 SimpleBar does not want to move. I have SimpleBar working on other map elements in IE11, but for some reason Layer List doesn't work. Here is a JSFiddle: https://jsfiddle.net/rxj5j7sf/37/
I appreciate any advice. Thank you.

**Comment by android-sos** [comment]
I'm getting the same problem with IE11, the scroll bar appear but it doesn't work.

**Comment by Grsmto** [comment]
Should be fixed in the latest version.

**Comment by arturkin** [comment]
Still have this problem with 2.5.1 - scrollbar is there but not moving at all.

**Comment by paullaros** [comment]
Issue still appearing on 2.5.1 in IE11

**Comment by arturkin** [comment]
I had the same problem. After the investigation, it seems that the problem is not with the plugin but with IE11 `flexbox` bugs, In some cases `.simplebar-scroll-content` height equals parent `flex` container height. And because of that, there are no scrollable element.

I found simple reproduction of this behavior:
https://jsbin.com/suyojonezu/1/edit?html,output

The solutions I've found:
* fix height of the parent
* wrap the parent with `display: flex` block and add `flex-basis: 100%` to the parent.
